samples_project/Assets/SampleViewer/Samples/Measure/Measure.unity

```diff
@@ -24,7 +24,7 @@ RenderSettings:
   m_AmbientEquatorColor: {r: 0.114, g: 0.125, b: 0.133, a: 1}
   m_AmbientGroundColor: {r: 0.047, g: 0.043, b: 0.035, a: 1}
   m_AmbientIntensity: 1
-  m_AmbientMode: 0
+  m_AmbientMode: 4
```

Comment:
I don't think Measure changes should go in the third person controller PR. Better to keep these changes atomic.

Reply:
Can we fix it in a separate PR? Bad practice to include in an unrelated PR.

Reply:
ok, yeah there're some other things that I can improve in the measure PR too so I can do them altogether